src/app/shared/dialogs/dialogs-list.service.ts

```diff
@@ -47,8 +49,8 @@ export class DialogsListService {
 
   getListAndColumns(db: string, selector?: any, planetField: string = 'local') {
     selector = selector || this.defaultSelectorFunctions()[db];
-    const fields = db === '_users' || db === 'child_users' ? this.userService.userProperties : [];
-
+    const fields = db === '_users' || db === 'child_users' ? this.couchService.post('_users/_find', { 'selector': { } }) : [];
```

Comment:
You should return a post to the `db + /_find` endpoint if `db === '_users'`.  Remember to include `this.userService.userProperties` as `fields` for that request.